src/packages/emmett/src/eventStore/subscriptions/streamingCoordinator.unit.spec.ts

```diff
@@ -159,10 +159,11 @@ const createMockEvent = (
   position: bigint,
 ): ReadEvent<MockEvent, ReadEventMetadataWithGlobalPosition> => ({
   type: 'Mocked',
+  kind: 'Event',
   data: { mocked: true },
   metadata: {
     streamName: 'testStream',
-    eventId: `event-${position}`,
+    messageId: `event-${position}`,
     streamPosition: position,
```

Comment:
Should "event" from value be changed to "message" too? 🤔

Reply:
I could do that too, indeed. 👍

Reply:
Did it 👍

Reply:
Did that ;)